graphql/types/Benefits.graphql

```diff
@@ -0,0 +1,30 @@
+type Benefits {
```

Comment:
You could remove the type `Benefits` and maintain only the type `Benefit`.

And, in the schema you would have: `benefits(...): [Benefit]`

Reply:
But its not an array of Benefits. Actually we have a field called ```ratesAndBenefitsData``` which contains the informations about the Benefit itself.